advanced/Scripts/piholeDebug.sh

```diff
@@ -1076,18 +1076,27 @@ show_db_entries() {
     IFS="$OLD_IFS"
 }
 
+show_groups() {
+    show_db_entries "Groups" "SELECT * FROM \"group\"" "4 100 7 10 13 50"
+}
+
 show_adlists() {
     show_db_entries "Adlists" "SELECT * FROM adlist" "4 100 7 10 13 50"
+    show_db_entries "Adlist groups" "SELECT * FROM adlist_by_group" "4 100 7 10 13 50"
```

Comment:
The column widths should be adjusted, as there are only two columns in this table.